src/views/profile/bookmarks-view/components/empty-state/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { IconBookmarkAdd16 } from '@hashicorp/flight-icons/svg-react/bookmark-add-16'
+import ButtonLink from 'components/button-link'
+import Heading from 'components/heading'
+import IconTile from 'components/icon-tile'
+import Text from 'components/text'
+import s from './empty-state.module.css'
+
+export default function BookmarksEmptyState() {
+	return (
+		<div className={s.wrapper}>
+			<IconTile size="small">
+				<IconBookmarkAdd16 />
+			</IconTile>
+			<Heading level={2} size={400} weight="semibold" className={s.heading}>
+				You have no saved bookmarks.
+			</Heading>
```

Comment:
I'm seeing [in Figma](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=12357%3A148760) that this should be size `300`.

![CleanShot 2022-08-23 at 11 18 08@2x](https://user-images.githubusercontent.com/43934258/186196822-e609a8f0-d1fb-4299-9259-17e5abed0613.png)

Reply:
Ah thank you! I always just look at the font size and forget that those naming helpers are there. Updated 😄